node/src/chain_spec/turing.rs

```diff
@@ -96,6 +96,13 @@ pub fn turing_development_config() -> ChainSpec {
 						5_376_000_000_000,
 						1_000_000_000,
 					),
+					(
+						2000,
+						turing_runtime::NATIVE_TOKEN_ID,
+						false,
+						5_376_000_000_000,
+						1_000_000_000,
+					),
```

Comment:
I supposed that this is a dev config. Do we want to add it?

Reply:
This is being added in the `turing_development_config` so it's fine to add as long as it's actually required for dev. If this is not required for ongoing development after this integration then it should not be merged.

Reply:
Got it 👍